projects/packages/jetpack-mu-wpcom/src/features/classic-profile-settings-notice/classic-profile-settings-notice.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * On the profile page, show a notice informing about profile settings on "Classic" admin-interface sites being "untangled" from `/me` and `/me/account`
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+/**
+ * Check whether the current page is the profile page
+ */
+function is_profile_page() {
```

Comment:
If we're keeping this function maybe we ought to have something prefixed more heavily.  `is_profile_page` is a bit generic and this function is in the global scope so we might need to be be cautious here - we don't know what functions other plugins will add to the global scope.

Perhaps something like `wpcom_is_profile_page` at a minimum

Reply:
I've removed it on https://github.com/Automattic/jetpack/pull/38134/commits/5522891d5e403343afe38001639d6fa6733415f9, as discussed with Bogdan.